servers/rest-services/src/test/java/org/projectnessie/services/rest/ContentKeyParamConverterProviderTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.services.rest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import javax.ws.rs.ext.ParamConverter;
+import org.junit.jupiter.api.Test;
+import org.projectnessie.model.ContentKey;
+
+public class ContentKeyParamConverterProviderTest {
+
+  private final ParamConverter<ContentKey> converter =
+      new ContentKeyParamConverterProvider().getConverter(ContentKey.class, null, null);
+
+  @Test
+  public void testNulls() {
+    assertThat(converter.fromString(null)).isNull();
+    assertThat(converter.toString(null)).isNull();
+  }
+
+  @Test
+  public void testValid() {
```

Comment:
Why "invalid"? The key value is supported, right? Maybre rename to `testDots`?

Reply:
it says `valid` :)

Reply:
Yeah, I see it not :)